bundles/org.openhab.binding.solax/README.md

```diff
@@ -111,6 +111,40 @@ If you're missing a channel this means that it's not supported for your inverter
 | serialNumber      | The serial number of the Wi-Fi module     |
 | inverterType      | Inverter Type (for example X1_HYBRID_G4)  |
 
+
+### Cloud Connect Inverter Configuration
+
+| Parameter         | Description                                                                                                                                        |
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| refreshInterval   | Defines the refresh interval when the binding polls from from the Solax cloud (in seconds). Optional parameter(min=15, max=600). Default is 30 seconds. Be advised that the cloud API is limited to max 10 calls per minute and 10000 calls per day.                                                                                                              |
```

Comment:
It's to be on the safe side so we don't reach it due to OH restarts or refresh commands (not very likely though)... I guess 9 or maybe 10 (because it's more natural value) can be used....

Reply:
In that case i would suggest to set the min to 9 as that is the absolute minimum. And set the default to 30 or so.